src/sentry/auth/authenticators/u2f.py

```diff
@@ -50,13 +50,11 @@ class U2fInterface(AuthenticatorInterface):
     type = 3
     interface_id = "u2f"
     configure_button = _("Configure")
-    name = _("U2F (Universal 2nd Factor)")
+    name = _("Passkey / Biometric / Authenticator")
     description = _(
-        "Authenticate with a U2F hardware device. This is a "
-        "device like a Yubikey or something similar which "
-        "supports FIDO's U2F specification. This also requires "
-        "a browser which supports this system (like Google "
-        "Chrome)."
+        "Authenticate using a Passkey, Biometrics, or a physical hardware "
+        "device such as a YubiKey. Requires a modern browser supporting the "
+        "WebAuthn standard."
```

Comment:
```suggestion
        "device such as a YubiKey. Requires a modern browser supporting the "
        "WebAuthn standard."
```

I don't think you need to mention modern browser. If they can view sentry, they have a compatible browser.

Reply:
Sounds good